src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/_Layout.cshtml

```diff
@@ -35,32 +35,33 @@
     <base href="~/" />
     <link rel="icon" href="~/favicon.ico" type="image/x-icon" asp-append-version="true">
-    <link rel="stylesheet" href="_content/Bit.BlazorUI/styles/bit.blazorui.min.css" asp-append-version="true" />
-    <link rel="stylesheet" href="Bit.BlazorUI.Playground.Web.styles.css" asp-append-version="true" />
-    <link rel="stylesheet" href="~/styles/styles.css" asp-append-version="true" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/styles/default.min.css">
+    <link media="all" rel="stylesheet" href="_content/Bit.BlazorUI/styles/bit.blazorui.min.css" asp-append-version="true" />
+    <link media="all" rel="stylesheet" href="Bit.BlazorUI.Playground.Web.styles.css" asp-append-version="true" />
+    <link media="all" rel="stylesheet" href="~/styles/styles.css" asp-append-version="true" />
+    <link media="all" rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/styles/default.min.css">
     <component type="typeof(HeadOutlet)" render-mode="@renderMode" />
 </head>
 <body>
 
     @RenderBody()
 
-    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/highlight.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/languages/csharp.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/languages/xml.min.js"></script>
-    <script src="_content/Bit.BlazorUI/scripts/bit.blazorui.min.js" asp-append-version="true"></script>
-    <script src="_content/Bit.BlazorUI.Charts/scripts/bit.blazorui.charts.min.js" asp-append-version="true"></script>
-    <script src="~/scripts/app.js" asp-append-version="true"></script>
+    <script defer src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/highlight.min.js"></script>
+    <script defer src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/languages/csharp.min.js"></script>
+    <script defer src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/languages/xml.min.js"></script>
+    <script async src="_content/Bit.BlazorUI/scripts/bit.blazorui.min.js" asp-append-version="true"></script>
+    <script async src="_content/Bit.BlazorUI.Charts/scripts/bit.blazorui.charts.min.js" asp-append-version="true"></script>
+    <script async src="~/scripts/app.js" asp-append-version="true"></script>
 
     @if (isBlazorServer)
     {
-        <script src="_framework/blazor.server.js" asp-append-version="true"></script>
+        <script async src="_framework/blazor.server.js" asp-append-version="true"></script>
     }
     else
     {
-        <script src="_framework/blazor.webassembly.js" asp-append-version="true"></script>
+        <script async src="_framework/blazor.webassembly.js" asp-append-version="true"></script>
```

Comment:
it's dangerous to use async for the main scripts of the project, it'll ignore the order of the scripts and may cause uncontrolled problems.
and since all scripts are registered at the end of the page the defer attribute is useless here.
please remove both async and defer attributes.